test/functional/launch/nested_direct/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+###############################################################################
+# Copyright (c) 2016-23, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+#
+# List of segment types for generating test files.
+#
+#  TODO: Support list segments with RAJA Launch
```

Comment:
Please capture items like this in GitHub issues so we don't forget about them.

Reply:
oops, this is an outdated comment we support list segments now